src/main/java/org/dependencytrack/persistence/ProjectQueryManager.java

```diff
@@ -518,8 +594,25 @@ public Project clone(
             final boolean includeACL,
             final boolean includePolicyViolations
     ) {
-        final var jsonMapper = new JsonMapper();
+        return clone(from, newVersion, includeTags, includeProperties, includeComponents, includeServices, includeAuditHistory,
+                includeACL, includePolicyViolations, false);
+    }
 
+    @Override
+    public Project clone(
+            final UUID from,
+            final String newVersion,
+            final boolean includeTags,
+            final boolean includeProperties,
+            final boolean includeComponents,
+            final boolean includeServices,
+            final boolean includeAuditHistory,
+            final boolean includeACL,
+            final boolean includePolicyViolations,
+            final boolean makeCloneLatest
+    ) {
```

Comment:
`makeCloneLatest = true` currently never reaches the cloning logic, because `CloneProjectTask` calls the old signature of this method: https://github.com/DependencyTrack/dependency-track/blob/7b183654fec6a5a6e07f5ba54b6f8f43496770cd/src/main/java/org/dependencytrack/tasks/CloneProjectTask.java#L49-L59

You should be able to remove the old method signature, since there are no callers outside `CloneProjectTask` anyway.

Would be good to have a regression test for this, you can find an example of project cloning tests here: https://github.com/DependencyTrack/dependency-track/blob/7b183654fec6a5a6e07f5ba54b6f8f43496770cd/src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java#L1477-L1522